pyxlsb/cell.py

```diff
@@ -13,19 +13,43 @@ def c(self):
     def v(self):
         return self.value
 
+    @property
+    def value_conv(self):
+        dtype = self.dtype
```

Comment:
Could you bring the type detection logic here. I would like to avoid the `dtype` and switching on arbitrary string because it makes the code more brittle and harder to refactor.

I know it won't be optimized, but this is ok for now.